### ISSUE TITLE
broken_barh docstring incorrect information

### ISSUE THREAD

**Issue by frankliuao** [matplotlib/matplotlib#13768]
### Correction recommendation

**Bug summary (not a bug but a doc error)**

https://matplotlib.org/api/_as_gen/matplotlib.pyplot.broken_barh.html?highlight=broken_barh#matplotlib.pyplot.broken_barh

**Error**
```
yranges : (*ymin*, *ymax*)
The y-position and extend for all the rectangles.
```
The text of the explanation is correct, however, the *ymax* should have been *ywidth*

**Matplotlib version**

  * Operating system: OSX
  * Matplotlib version: 3.0.3
  * Matplotlib backend (`print(matplotlib.get_backend())`): module://ipykernel.pylab.backend_inline

  * Python version: '3.7.2 (default, Dec 29 2018, 00:00:04) \n[Clang 4.0.1 (tags/RELEASE_401/final)]'
  * Jupyter version (if applicable): 4.4.0
  * Other libraries: 

conda base

**Comment by NicolasCourtemancheAtMcGill** [comment]
Hello,
Upon examining the type of plot `broken_barh` produces, it seems your observation is correct. I will look into changing the variable name in the code and docstring and will open a pull request.

**Comment by frankliuao** [comment]
> Hello,
> Upon examining the type of plot `broken_barh` produces, it seems your observation is correct. I will look into changing the variable name in the code and docstring and will open a pull request.

Yes I've already created a PR.

**Comment by timhoffm** [comment]
Closed in #13770.
